src/ntfs/read.rs

```diff
@@ -668,6 +668,19 @@ where
     Ok(())
 }
 
+pub(crate) fn manage_occupancies<H>(
+    collections: &mut Collections,
+    file_handler: &mut H,
+) -> Result<()>
+where
+    for<'a> &'a mut H: FileHandler,
+{
+    let file = "occupancies.txt";
+    let occupancies = read_objects_loose::<_, Occupancy>(file_handler, file, false)?;
+    collections.occupancies = Collection::new(occupancies);
+    Ok(())
```

Comment:
It might be relevant to check the memory build like here :thinking: 
https://github.com/hove-io/transit_model/blob/a3649520def97797ee3fff92a572aab72ddc6b90/src/ntfs/read.rs#L626
on 3 main fields (line_id, from_stop_area, to_stop_area)

Reply:
I actually started doing exactly what you have in mind, but I realized that we're going to do that on `sanitize` anyway and that it might be better to do it only once. No strong opinion on either solution. Do you prefer me to do it on `manage_occupancies` too?

Reply:
If it's not too long, do it. It seems more elegant to me to clean beforehand.
Otherwise let it go.